index.js

```diff
@@ -98,6 +100,44 @@ app.post("/github_discord_urls", async (req, res) => {
   }
 });
 
+async function get_user_details_from_db(email) {
```

Comment:
rename: get_<table name>_by_<params you're using>
so in this case, it would be: get_auth_user_info_by_email